package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@digicatapult/vitalam-ipfs",
+  "version": "1.0.0",
+  "description": "Service for WASP",
+  "main": "app/index.js",
+  "scripts": {
+    "test": "NODE_ENV=test mocha --config ./test/mocharc.js ./test",
+    "test:integration": "NODE_ENV=test mocha --config ./test/mocharc.js ./test/integration",
+    "test:unit": "NODE_ENV=test mocha --config ./test/mocharc-unit.js ./test",
+    "lint": "eslint .",
+    "depcheck": "depcheck",
+    "start": "node app/index.js",
+    "dev": "NODE_ENV=dev nodemon app/index.js | pino-colada",
+    "coverage": "nyc npm run test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digicatapult/vitalam-ipfs.git"
+  },
+  "engines": {
+    "node": ">=16.13.0"
```

Comment:
We should probably have another ticket in the backlog for updating the `vitalam-service-template` repo to node `v16.x`

Reply:
👍  I've already put a ticket in our Jira backlog for that IN-74